lua/neotest-rspec/init.lua

```diff
@@ -90,16 +97,6 @@ function NeotestAdapter.build_spec(args)
     table.insert(script_args, position.path)
   end
 
-  local function run_by_test_name()
```

Comment:
lmk how you feel about this, it wasn't getting used. kinda forgot I got rid of it